dds_web/database/models.py

```diff
@@ -274,8 +274,11 @@ class User(flask_login.UserMixin, db.Model):
 
     # Relationships
     identifiers = db.relationship("Identifier", back_populates="user", passive_deletes=True)
-    emails = db.relationship("Email", back_populates="user", passive_deletes=True)
+    emails = db.relationship("Email", back_populates="user", passive_deletes=True, cascade="all")
```

Comment:
I think this was not needed after all, or am I confusing it with a different relationship?

Reply:
No you're remembering correctly. This is not needed.